personality/xrootd/env.pan

```diff
@@ -35,3 +38,19 @@ variable XROOTD_MONITORING_DESTINATIONS ?= undef;
 variable XROOTD_REPORTING_DESTINATIONS ?= undef;
 
 
+# Default GSI-related parameters
+variable XROOTD_SECURITY_GSI_PARAMS ?= {
+  SELF["ca"] = 2;
+  SELF["cert"] = "/etc/grid-security/dpmmgr/dpmcert.pem";
+  SELF["crl"] = 3;
+  SELF["gmapopt"] = 10;
+  SELF["key"] = "/etc/grid-security/dpmmgr/dpmkey.pem";
+  SELF["md"] = "sha256:sha1";
+  if ( VOMS_XROOTD_EXTENSION_ENABLED ) { 
+    SELF["vomsfun"] = "/usr/lib64/libXrdSecgsiVOMS.so";
+  } else {
+    SELF["vomsat"] = 0;
+  };
+
+  SELF;
+};
```

Comment:
I find these blocks of DML that use SELF to build dicts hard to follow, what is your opinion of the following equivalent?

```pan
variable XROOTD_SECURITY_GSI_PARAMS ?= dict(
  "ca", 2,
  "cert", "/etc/grid-security/dpmmgr/dpmcert.pem",
  "crl", 3,
  "gmapopt", 10,
  "key", "/etc/grid-security/dpmmgr/dpmkey.pem",
  "md", "sha256:sha1",
  if (VOMS_XROOTD_EXTENSION_ENABLED) "vomsfun" else "vomsat",
  if (VOMS_XROOTD_EXTENSION_ENABLED) "/usr/lib64/libXrdSecgsiVOMS.so" else 0,
);
```